earth2mip/forecasts.py

```diff
@@ -70,6 +70,25 @@ async def __getitem__(self, i: int):
             yield x
 
 
+class select_channels(Forecast):
```

Comment:
Odd class name... lower case. If this is kept I think it needs to get changed.

Reply:
I dont particularly like this class, feels like a band aid on a bigger problem with the forecast. Granted I dont have a better solution myself for what I understand you want to do.

Reply:
I made the name lower case so that it looks like a function:
```
regrided_forecast = select_channels(forecast, ["z500"])
scores = some_scoring_workflow(regridded_forecast)
```

A lazy data structure with transformations is a common pattern in data analysis. The idea is that we can transform whole Forecasts objects in a lazy manner similar to xarray.Dataset, a spark RDD, torch DataPipe. 

I agree the Forecast concept may need more thought...it's basically just used by lagged ensembles right now, and I haven't decided whether more workflows should be written using it.

Reply:
@NickGeneva perhaps you can clarify in a separate document what your issues with the forecast object are. This pr does not introduce or change this api, so I think this discussion is out of scope.

Reply:
Got it, thanks for clarifying.